task/buildah/0.1/buildah.yaml

```diff
@@ -70,7 +70,7 @@ spec:
     - name: DOCKER_CONFIG
       value: $(params.DOCKER_AUTH)
   steps:
-  - image: $(params.BUILDER_IMAGE)
+  - image: quay.io/redhat-appstudio/buildah:v1.28
```

Comment:
Should we change the default image in the BUILDER_IMAGE parameter instead of here? It seems like the `prototype-build-compliance` pipeline is still setting this parameter.

In case we want to force the use of buildah:v1.28, we should get rid of the BUILDER_IMAGE param altogether.

Reply:
`prototype-build-compliance` can be ignored. The BUILDER_IMAGE is used in next steps of this task since new image requires additional "SYS_CHROOT" capabality or running in `unshare`, so I kept old image for other tasks, since we need the 1.28 only for build step.